e2e/launcher/network.go

```diff
@@ -0,0 +1,43 @@
+package launcher
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+type Network struct {
+	Nodes                []*Node
+	NetworkConfiguration *NetworkConfiguration
+}
+
+func LaunchNetwork(networkConfigDirectory string, executablePath string) (network Network, err error) {
```

Comment:
Decision was possibly made because members of the struct already point at things. I will revisit this line of thought and possibly be better able to articulate why I decide whatever I end up with.